attestation/context.go

```diff
@@ -33,6 +33,7 @@ const (
 	ExecuteRunType     RunType = "execute"
 	ProductRunType     RunType = "product"
 	PostProductRunType RunType = "postproduct"
+	VerifyRunType      RunType = "verify"
```

Comment:
I've changed `policyverify` to to have a `VerifyRunType`. Currently, you can actually perform `run -a=policyverify`, which of course fails as it isn't correctly scaffolded in the CLI. By adding this run type there is logic (here) that lets us ensure that it isn't used in tandem with other types.